install.sh

```diff
@@ -166,8 +166,8 @@ for arch in x86 x64 arm arm64; do
 done
 rm msvcenv.sh
 
-if [ -d "$DEST/bin/$host" ] && [ -x "$(which wine64 2>/dev/null)" ]; then
-    WINEDEBUG=-all wine64 wineboot &>/dev/null
+if [ -d "$DEST/bin/$host" ]; then
+    WINEDEBUG=-all $(command -v wine64 || command -v wine || false) wineboot &>/dev/null
```

Comment:
This change isn't right; if there's no wine installed, this will end up trying to still execute something and fail.

This breaks in the clang/lld testcase in CI, where we explicitly don't install wine and don't want to use that, we're just going to use the MSVC headers/libraries with clang/lld.

Reply:
Ah, I was executing "false" purposely, so if wine64 or wine wasn't installed, it would run "false" to explicitly cause an error here, assuming that not having wine installed should be fatal.  I'll fix this up.

Reply:
I am assuming this is the only place this would be needed - if not let me know.

I also didn't change the README file - I don't think you intended to be that pedantic since that's human readable docs.  I'll make these changes now.